CONTRIBUTING.md

```diff
@@ -145,6 +145,15 @@ When developing a new feature, add new tests to cover all code paths.
 [<Test>]
 let ``preserve compile directive between piped functions, 512`` () = ...
 ```
+- Give your PR a meaningful title. Make sure it covers the change you are introducing in Fantomas.
+
+    For example:
+*"Fix bug 1404"* is a poor title as it does not tell the maintainers what changed to the codebase.<br />
+*"Don't double unindent when record has access modifier"* is better as informs what exactly has changed.
+- Add a link to the issue you are solved by using [a keyword](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) in the PR description.<br />
+*"Fixes #1404"* does the trick quite well.
+- Not mandatory but when fixing a bug, consider using the `fix-issue-number` as git branch name.<br />
```

Comment:
```suggestion
- Not mandatory, but when fixing a bug consider using the `fix-issue-number` as git branch name.<br />
```
Gah, commas, so complicated!  I'm pretty sure the first one after "mandatory" is correct as an introductory clause.  I /think/ the second can be removed because "when fixing a bug" is an incomplete sentence by itself.  Unless, of course, you consider this a parenthetical clause or even a second introductory clause?  Eh, it's close.  But I think just take it out.